src/Common/Models/RobotModel.cs

```diff
@@ -155,4 +155,63 @@ public bool IsValid()
         }
         return true;
     }
+    /// <summary>
+    /// Set the topic to be enabled.
+    /// </summary>
+    /// <param name="topic"></param>
+    public void EnableRosTopic(RosTopicModel topic)
+    {
+        topic.Enabled = true;
+    }
+
+    /// <summary>
+    /// Set the topic to be disabled.
+    /// </summary>
+    /// <param name="topic"></param>
+    public void DisableRosTopic(RosTopicModel topic)
```

Comment:
Move function to `RosTopicModel` name it `Disable()`.

Reply:
renamed and moved to RosTopicModel